Lombiq.HelpfulLibraries.AspNetCore/Security/CdnContentSecurityPolicyProvider.cs

```diff
@@ -52,7 +54,10 @@ public class CdnContentSecurityPolicyProvider : IContentSecurityPolicyProvider
     /// <summary>
     /// Gets the sources that will be added to the <see cref="FrameSrc"/> directive.
     /// </summary>
-    public static ConcurrentBag<string> PermittedFrameSources { get; } = [];
+    public static ConcurrentBag<string> PermittedFrameSources { get; } = new(
+    [
+        "giscus.app", // #spell-check-ignore-line
+    ]);
```

Comment:
This is not a widely used CDN. Add an `IContentSecurityPolicyProvider` in the module instead, like https://github.com/Lombiq/Orchard-Azure-Application-Insights/blob/dev/Lombiq.Hosting.Azure.ApplicationInsights/Services/ApplicationInsightsContentSecurityPolicyProvider.cs

Reply:
Alright, I close this PR. Thanks!